sympy/integrals/tests/test_manual.py

```diff
@@ -474,3 +474,17 @@ def test_issue_15471():
     f = log(x)*cos(log(x))/x**(S(3)/4)
     F = -128*x**(1/4)*sin(log(x))/289 + 240*x**(1/4)*cos(log(x))/289 + (16*x**(1/4)*sin(log(x))/17 + 4*x**(1/4)*cos(log(x))/17)*log(x)
     assert manualintegrate(f, x) == F and F.diff(x).equals(f)
+
+
+def test_issue15494():
+    from sympy.abc import t, s
+
+    Z0 = Function('Z0')
+    Z1 = Function('Z1')
+
+    k01, k10= symbols('k01 k10', real=True, positive=True)
+
+    integrand = (exp(s/2)-2*exp(1.6*s)+exp(s))*exp(s)
```

Comment:
The result will be more exact if the float 1.6 is replaced by fraction:

    integrand = (exp(s/2) - 2*exp(8*s/5) + exp(s))*exp(s)

(Note the spaces round `-` and `+`.)

Reply:
The issue only comes up when numbers aren't given as fractions. I don't know if this makes the fix not really useful. I think it's good to fix it anyways since  Sympy can compute the integral even when 1.6 is given as a float. This is quite problematic when making the assertion in the test file though.

Reply:
This seems to indicate that issue  #15494 is not related to this fix in manualintegrate. In fact, I get on current master (without this patch):
```
>>> s = symbols('s')
>>> integrand = (exp(s/2)-2*exp(1.6*s)+exp(s))*exp(s)
>>> integrate(integrand, s)
0.666666666666667*exp(3*s/2) + 0.5*exp(2*s) - 0.769230769230769*exp(2.6*s)
```
So it looks like #15494 was fixed already.

Hence another test would be needed for the fix in this PR.

Reply:
Yeah I figured that this morning. Oddly enough, and this points to the fact that the issue is indeed probably unrelated to this fix, you need to define the symbol with the "real=True" and/or "positive=True" for the issue to appear.